packages/react-native/platforms/ios/Podfile

```
@@ -0,0 +1,5 @@
+platform :ios, '13.0'
+
+pod 'React-Core', path: '../../node_modules/@ammarahm-ed/react-native/React-Core.podspec'
+pod 'React', path: '../../node_modules/@ammarahm-ed/react-native/React.podspec'
```

We want to avoid digging down into `node_modules` as it may fall apart in non-standard repo structures like monorepos. If we move `React-Core.podspec` and `React.podspec` to be alongside this `Podfile`, the paths can become:

```suggestion
pod 'React-Core', path: 'React-Core.podspec'
pod 'React', path: 'React.podspec'
```

(Might need a `./` at the start of the path, but probably not).

---

This did not work since the Podfile is merged into the app's podfile, it needs the actual path to the file & NativeScript does not resolve pods. A better way which I found on expo monorepo was to do this instead which is working:

```ruby
pod 'React-Core', path: File.join(File.dirname(`node --print "require.resolve('@ammarahm-ed/react-native/package.json')"`), "platforms/ios/React-Core.podspec")
```
Since it's not a relative path, we are just using node to get us the module, no matter where it is & get our podspec path.

---

Great find on that Expo trick; that's perfect!